### ISSUE TITLE
[BUG] no downloads available when game details request fails

### ISSUE THREAD

**Issue by mike14160** [hydralauncher/hydra#526]
When I go and download a game, the game does not start to download, and when I click to download again appears that there is "no downloads available" and "Could not retrieve shop details"
<img width="900" alt="Screenshot 2024-05-27 09194" src="https://github.com/hydralauncher/hydra/assets/170557766/750af4b6-6864-45ba-be8b-00b6c2e279c3">

**Comment by Ferari1** [comment]
![image_2024-05-29_183607067](https://github.com/hydralauncher/hydra/assets/15855338/1d602ee2-39fe-4cfe-83e1-d16be2105336)
Same here. Fresh install of the app v1.2.4

**Comment by mike14160** [comment]
I installed hydra again, and the same problem continues to happen

**Comment by bracinhol2** [comment]
> Instalei o Hydra novamente e o mesmo problema continua acontecendo

cmg tbm ja tentei ate outras versão

**Comment by zamitto** [comment]
Could you attach your logs here? They are located in the same directory as your Hydra.exe file.
They are the files `combined.log`, `èrror.log` and `info.log`.

If your Hydra has any error when fetching the game page we should see it in the error file

**Comment by mike14160** [comment]
Sure, here they are 
[combined.log](https://github.com/user-attachments/files/15520920/combined.log)
[error.log](https://github.com/user-attachments/files/15520922/error.log)
[info.log](https://github.com/user-attachments/files/15520923/info.log)

**Comment by zamitto** [comment]
Thank you for the logs, got what the issue is. We will fix it for the next version

**Comment by mike14160** [comment]
Ok thank you, and do you know when the new version will be released?

**Comment by Richwrs** [comment]
Eu consegui identificar o que ocorreu no meu caso.
Quando o jogo é adicionado na Biblioteca, ele não mostra listas de download. Pra baixar basta remover da lista e aparecerá as opções.

> **I was able to identify what happened in my case.
> When the game is added to the Library, it does not show download lists. To download, simply remove it from the Library List and the download options will appear.**

**Comment by JoaoScheleder** [comment]
I'm having the same issue, 
> [!IMPORTANT]
> Fresh and first time install, version **2.0.1**

:electron: error.log
```
[2024-06-22 23:21:18.434] [error] (main) user is not logged in
[2024-06-22 23:21:18.437] [error]        Unhandled rejection Error: user is not logged in
    at HydraApi.revalidateAccessTokenIfExpired (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:932:13)
    at HydraApi.post (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:982:16)
    at createGame (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/main-BxyCXgQh.js:64:21)
    at file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index-Cbv99xph.js:314:9
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:82807)
[2024-06-22 23:27:20.499] [error]        Unhandled TypeError: Invalid URL
    at new URL (node:internal/url:775:36)
    at handleDeepLinkPath (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4908:17)
    at App.<anonymous> (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4920:5)
    at App.emit (node:events:518:28)
[2024-06-22 23:35:54.298] [error]        Unhandled TypeError: Invalid URL
    at new URL (node:internal/url:775:36)
    at handleDeepLinkPath (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4908:17)
    at App.<anonymous> (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4920:5)
    at App.emit (node:events:518:28)
[2024-06-22 23:38:50.675] [error]        Unhandled TypeError: Invalid URL
    at new URL (node:internal/url:775:36)
    at handleDeepLinkPath (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4908:17)
    at App.<anonymous> (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4920:5)
    at App.emit (node:events:518:28)
[2024-06-22 23:41:37.500] [error]        Unhandled TypeError: Invalid URL
    at new URL (node:internal/url:775:36)
    at handleDeepLinkPath (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4908:17)
    at App.<anonymous> (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4920:5)
    at App.emit (node:events:518:28)
[2024-06-22 23:43:21.184] [error]        Unhandled TypeError: Invalid URL
    at new URL (node:internal/url:775:36)
    at handleDeepLinkPath (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4908:17)
    at App.<anonymous> (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4920:5)
    at App.emit (node:events:518:28)
[2024-06-22 23:43:48.174] [error]        Unhandled TypeError: Invalid URL
    at new URL (node:internal/url:775:36)
    at handleDeepLinkPath (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4908:17)
    at App.<anonymous> (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4920:5)
    at App.emit (node:events:518:28)
[2024-06-22 23:45:12.990] [error]        Unhandled TypeError: Invalid URL
    at new URL (node:internal/url:775:36)
    at handleDeepLinkPath (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4908:17)
    at App.<anonymous> (file:///C:/Users/user/AppData/Local/Programs/Hydra/resources/app.asar/out/main/index.js:4920:5)
    at App.emit (node:events:518:28)
```

**Comment by zamitto** [comment]
This should be fixed now in version 2.0
